serviceWrappers/linux/kLiveController

```diff
@@ -21,48 +21,71 @@
 ###
 # extend callable functions from "functions" script
 ###
+NAME=kLiveController
+LOG_DIR=/var/log
```

Comment:
LOG_DIR should not be a literal, plus, it is reassigned in line 56. PID_DIR should also not be a literal, it is not reassigned later, though.